ipfs-api/src/response/dag.rs

```diff
@@ -29,9 +29,15 @@ pub struct DagGetResponse {
 }
 
 #[derive(Debug, Deserialize)]
-#[serde(rename_all = "PascalCase")]
 pub struct DagPutResponse {
-    pub cid: String,
+    #[serde(rename = "Cid")]
+    pub cid: Cid,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct Cid {
+    #[serde(rename = "/")]
```

Comment:
Is this always guaranteed to be named this? Looking at the docs, it seems like it is, but I figured I'd ask since I know so little about this route

https://docs.ipfs.io/reference/http/api/#api-v0-dag-put

Reply:
I assumed the docs were correct.